src/olympia/devhub/templates/devhub/new-landing/components/banner.html

```diff
@@ -10,7 +10,7 @@ <h2>{{ _('Build best-in-class extensions and themes') }}</h2>
             we’ve got the resources to support you.
           {% endtrans %}
         </p>
-        <a href="{{ settings.EXTENSION_WORKSHOP_URL }}">{{ _('Visit the Extension Workshop') }}</a>
+        <a href=" https://extensionworkshop.com?utm_source=addons.mozilla.org&utm_medium=devhub&utm_content=banner/">{{ _('Visit the Firefox Extension Workshop') }}</a>
```

Comment:
can you (re-)replace `https://extensionworkshop.com` with `{{ settings.EXTENSION_WORKSHOP_URL }}\`

Reply:
oh, and `utm_content=banner/` should be just `utm_content=banner`

Reply:
Hey, I may be wrong but just wanted to point out that in `{{ settings.EXTENSION_WORKSHOP_URL }}\ ` there is a `\` instead of `/`.

Reply:
`/`  (I wrote the wrong slash.)

Reply:
Thanks for the confirmation @eviljeff. I think I've made all the necessary changes in my last 2 commits. Please review it and let me know if any other changes are required.